jerry-core/debugger/debugger.h

```diff
@@ -376,6 +376,7 @@ typedef struct
 typedef struct
 {
   uint8_t type; /**< type of the message */
+  uint8_t min_depth[sizeof (uint32_t)]; /**< minimum depth*/
```

Comment:
Why don't we use `uint16_t` for both `min_depth` and `max_depth`? @zherczeg, what do you think?

Reply:
Currently only uint8, cpointer, and uint32 types are used by the protocol. Adding new types would require adding new encoders/decoders for clients. This is certainly possible, but for this it does not seem worth the effort.

Reply:
I see. Thanks for the explanation.